examples/servicegraph/docker/BUILD

```diff
@@ -0,0 +1,35 @@
+load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar")
```

Comment:
can we add to `mixer/docker/gcloud_build.sh` (or put a new script here) that would: `bazel run  //examples/servicegraph/docker:servicegraph gcr.io/$PROJECT/servicegraph:$DOCKER_TAG` (and also tag with `latest` and push via gcloud)?